### PR TITLE
ROECCT-115 Important Banner

### PR DESCRIPTION
**Pull request by kullas-ch**
### JIRA link
https://companieshouse.atlassian.net/browse/ROECCT-115

### Work checklist

- [ ] Tests added where applicable
- [ ] UI changes meet accessibility criteria

### Merge instructions

We are committed to keeping commit history clean, consistent and linear. To achieve this, this commit should be structured as follows:

```
<type>[optional scope]: <description>
```

and contain the following structural elements:

- fix: a commit that patches a bug in your codebase (this correlates with PATCH in semantic versioning),
- feat: a commit that introduces a new feature to the codebase (this correlates with MINOR in semantic versioning),
- BREAKING CHANGE: a commit that has a footer `BREAKING CHANGE:` introduces a breaking API change (correlating with MAJOR in semantic versioning). A BREAKING CHANGE can be part of commits of any type,
- types other than `fix:` and `feat:` are allowed, for example `build:`, `chore:`, `ci:`, `docs:`, `style:`, `refactor:`, `perf:`, `test:`, and others,
- footers other than `BREAKING CHANGE: <description>` may be provided.

### CODE REVIEW

File: views/includes/page-templates/important-banner.html [hunk 1]
```diff
@@ -0,0 +1,10 @@
+{% from "govuk/components/notification-banner/macro.njk" import govukNotificationBanner %}
+{% set html %}
+
+You need to provide some information about the overseas entity from the pre-registration period <br></br>
+{% include "includes/page-templates/relevant-period.html" %}
```
Comment (ConnorOS-ch) [inline views/includes/page-templates/important-banner.html:5]: could we enclose this text in a `<p>` element. You wont need to use `<br>` in that case also

File: views/includes/page-templates/relevant-period.html [hunk 2]
```diff
@@ -0,0 +1,6 @@
+{% set start_date = "28 February 2022" %}
+{% set registration_date = dateMacros.formatDate(dateOfCreation["day"], dateOfCreation["month"], dateOfCreation["year"]) %}
+
+<p class="govuk-body-m">
```
Comment (ConnorOS-ch) [inline views/includes/page-templates/relevant-period.html:4]: I don't think `class="govuk-body-m"` is doing anything so safe to remove

**Review by amalviyach — APPROVED** [review]

File: views/includes/page-templates/important-banner.html
Comment (ConnorOS-ch) [inline views/includes/page-templates/important-banner.html:1]: I don't know is includes/page-templates is the correct place to put this file and relevant-period.html

maybe just includes is better

**Review by ConnorOS-ch — CHANGES_REQUESTED** [review]
A few stylistic comments. Looks great though

File: views/includes/page-templates/important-banner.html
Comment (mzacharias-ch) [reply views/includes/page-templates/important-banner.html:1]: it is in correct path it is a page template only and not a whole page.

**Review by mzacharias-ch — APPROVED** [review]

File: views/includes/page-templates/relevant-period.html (hunk 2 above)
Comment (mzacharias-ch) [reply views/includes/page-templates/relevant-period.html]: that class is doing margin on bottom 20px just like is in describe in prototype. Please check prototype and you will see they are using this class.

**Review by ConnorOS-ch — APPROVED** [review]

File: views/includes/page-templates/important-banner.html
Comment (ConnorOS-ch) [reply views/includes/page-templates/important-banner.html:1]: Page-templates is used when a whole or most of a page content is reused between similar pages.

This is an individual component not a page template it isn't similar in use to the files already in page-templates

**Pull request merged**